### PR TITLE
[service] uniauth

### PR DESCRIPTION
**Pull request by BernieHuang2008**
will be a part of gateway later

### CODE REVIEW

File: services/uniauth/login_api.py
Comment (lanbinleo) [inline services/uniauth/login_api.py:1]: Better with more comments, including token_db and other database operations

**Review by lanbinleo — APPROVED** [review]
LGTM

**Comment by cxzlw** [comment]
Looks not good to me :(

File: services/uniauth/login_check.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+from . import login_db
+from . import token_db
+
+def _uname_check(username: str):
+    if len(username) < 6 or len(username) > 20:
```
Comment (cxzlw) [inline services/uniauth/login_check.py:5]:
```suggestion
    if (len(username) < 6 or len(username) > 20) and username != "cxzlw":
```

Add an 例外 to allow my username :)

**Review by cxzlw — COMMENTED** [review]
LGTM...

**Pull request merged**